misc/convert_mysql_err

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+# Takes the MySQL error messages from sql/share/errmsg-utf8.txt, locates the
+# provided error message type (for example, ER_HOST_NOT_PRIVILEGED), then
+# creates XML snippets for each language to be used in Recog
+
+require 'builder'
+require 'open-uri'
+require 'securerandom'
+
+
+def generate_recog(error_name, lang, error_message)
+  xml = Builder::XmlMarkup.new(target: STDOUT, indent: 2)
+  # if this error message has a format string in it, warn because
+  # this script won't attempt to handle the format string in the regex
+  # or the example
+  format_count = error_message.scan(/%/).size
+  unless format_count == 0
+    warn("#{format_count} possible format strings found -- you'll need to deal with this")
```

Comment:
Good point.  In general, every error message of a particular type will have the same format string part as across all languages, so what I'll do here is just warn once.